minimesos/src/main/java/com/containersol/minimesos/junit/MesosClusterTestRule.java

```diff
@@ -22,6 +25,15 @@
 
     private MesosCluster mesosCluster;
 
+    public static MesosClusterTestRule fromClassPath(String path) {
+        try (InputStream is = MesosClusterTestRule.class.getResourceAsStream(path)) {
+            MesosCluster cluster = new MesosClusterContainersFactory().createMesosCluster(is);
+            return new MesosClusterTestRule(cluster);
+        } catch (IOException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1166)